.travis.yml

```diff
@@ -25,41 +43,74 @@ install:
   - conda config --append channels conda-forge
   # Useful for debugging any issues with conda
   - conda info -a
-  # Test conda build
-  - conda build -q conda-recipe --python $TRAVIS_PYTHON_VERSION --output-folder bld-dir
-  - conda config --add channels "file://`pwd`/bld-dir"
-  #Grab all dependencies
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pcdsdevices --file dev-requirements.txt
-  #Launch Conda environment
-  - source activate test-environment
+  # Test conda build and create test environment
+  - |
+    if [[ $UNIT_TEST || $BUILD_DOCS ]]; then
+      echo "Building full environment"
+      conda build -q conda-recipe --python $TRAVIS_PYTHON_VERSION --output-folder bld-dir
+      conda config --add channels "file://`pwd`/bld-dir"
+      conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pcdsdevices --file dev-requirements.txt
+    elif [[ $STYLE ]]; then
+      echo "Building minimal flake8 environment"
+      conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION flake8
```

Comment:
I think we can assume python + pip will be available in all of our travis matrices, so using conda isn't even required.

Reply:
I agree, when I read through your PR I realized that I had forgotten that pip exists.